test/functional/wallet_basic.py

```diff
@@ -338,6 +391,74 @@ def run_test(self):
         # This will raise an exception for importing an invalid pubkey
         assert_raises_rpc_error(-5, "Pubkey is not a valid public key", self.nodes[0].importpubkey, "5361746f736869204e616b616d6f746f")
 
+        # send with explicit btc/kb fee
+        self.sync_all(self.nodes[0:3])
+        self.log.info("test explicit fee (sendtoaddress as btc/kb)")
+        self.nodes[0].generate(1)
```

Comment:
The `sync_all` needs to be moved after `generate`, or `prebalance` could end up with a stale balance and cause the check below (line 426) to fail.

Reply:
Moved.